Makefile

```diff
@@ -158,9 +158,19 @@ check-docs: $(EMBEDMD) $(LICHE) build
 	@$(LICHE) --recursive docs --exclude "cloud.tencent.com" --document-root .
 	@$(LICHE) --exclude "cloud.tencent.com|goreportcard.com" --document-root . *.md
 
+# checks Go code comments if they have trailing period (excludes protobuffers and vendor files).
```

Comment:
Worth to mention that we are more strict in Thanos and requires that.

Plus I think if you want this special case for `// ` with spaces it should mentioned. However TBH I would drop it (: I restrict comments to be full sentence always...

Reply:
True that it should be mentioned if it stays :+1: 

I'm bit afraid to drop it. In case there was some issue with the regular then it will be easy to workaround it without blocking unrelated PRs until it's fixed since this is not that critical. 
Also I'm not sure about the lists and the edge case with testing. 

But if you say so, I can remove it.

Where would you announce this change? Is slack enough or put it to the changelog?

Reply:
>  then it will be easy to work around it without blocking unrelated PRs until it's fixed since this is not that critical.

It's always that with linting. But you can always workaround by dropping check temporarily. 

I would only mention it in thanos-dev channel. Not even CHANGELOG. It's only affecting code readability and documenting code by comments. (: so it should be fine. If it would be too strict we can always loosen this.

> Also I'm not sure about the lists and the edge case with testing.

Agree, but let's be opinionated in order to give readability.